projects/packages/search/src/class-options.php

```diff
@@ -48,6 +48,7 @@ class Options {
 	const OVERLAY_TRIGGER_IMMEDIATE = 'immediate';
 	const OVERLAY_TRIGGER_RESULTS   = 'results';
 	const OVERLAY_TRIGGER_SUBMIT    = 'submit';
+	const DEFAULT_OVERLAY_TRIGGER   = self::OVERLAY_TRIGGER_SUBMIT;
```

Comment:
I was unsure about using another class constant to define this one, but it looks like it's all good: 

https://wiki.php.net/rfc/const_scalar_exprs